tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_HalfSpaceMirror.cpp

```diff
@@ -62,44 +60,61 @@ struct HalfSpaceMirrorProxy : Elasticity::Solutions::HalfSpaceMirror {
 SPECTRE_TEST_CASE(
     "Unit.PointwiseFunctions.AnalyticSolutions.Elasticity.HalfSpaceMirror",
     "[PointwiseFunctions][Unit][Elasticity]") {
-  const size_t dim = Elasticity::Solutions::HalfSpaceMirror::dim;
+  const size_t dim = Elasticity::Solutions::HalfSpaceMirror::volume_dim;
   const Elasticity::Solutions::HalfSpaceMirror check_solution{
-      0.177, 1.0,
-      Elasticity::ConstitutiveRelations::IsotropicHomogeneous<3>{
-          // Iron: E=72, nu=0.17
-          36.36363636363637, 30.76923076923077},
-      50, 1.e-13};
+      0.177, Elasticity::ConstitutiveRelations::IsotropicHomogeneous<3>{
+                 // fused silica: E=72, nu=0.17
+                 36.36363636363637, 30.76923076923077}};
   const auto created_solution =
       TestHelpers::test_creation<Elasticity::Solutions::HalfSpaceMirror>(
           "BeamWidth: 0.177\n"
-          "AppliedForce: 1.0\n"
           "Material:\n"
           "  BulkModulus: 36.36363636363637\n"
-          "  ShearModulus: 30.76923076923077\n"
-          "IntegrationIntervals: 20\n"
-          "IntergrationTolerance: 1.e-13\n");
+          "  ShearModulus: 30.76923076923077\n");
   CHECK(created_solution == check_solution);
   test_serialization(check_solution);
 
   pypp::SetupLocalPythonEnvironment local_python_env{"PointwiseFunctions"};
   const Elasticity::ConstitutiveRelations::IsotropicHomogeneous<dim>
       constitutive_relation{36.36363636363637, 30.76923076923077};
-  const HalfSpaceMirrorProxy solution{0.177, 1.0, constitutive_relation, 100,
-                                      1.e-13};
+  const HalfSpaceMirrorProxy solution{0.177, constitutive_relation, 350};
```

Comment:
If you need a workspace size of 350 for a radius of ~100x beam width to work then perhaps the 350 should be the default. I have no idea how memory intensive that is, but it's probably not significant.